synthtool/gcp/common.py

```diff
@@ -269,6 +270,7 @@ def java_library(self, **kwargs) -> Path:
 
     def node_library(self, **kwargs) -> Path:
         # TODO: once we've migrated all Node.js repos to either having
+        metadata.start_tracking_generated_files()
```

Comment:
~Calling `start_tracking_generated_files()` twice won't cause any issues? (_we will call it once for each API version._)~

_question answered below._